src/components/SignupForm.js

```diff
@@ -135,13 +137,29 @@ class SignupForm extends Component {
                 }
               />
             </Form.Field>
-            <Button id="submit-account-button" type="submit" >
+
+            {this.state.role === "research_group" ? (
+              <Form.Field>
+                <label>Registration Key</label>
+                <input
+                  id="subscription-key"
```

Comment:
Two different terms are used to describe the same thing in the label and the input id.

Reply:
Good catch, fixed